test/fix-me.js

```diff
@@ -1,5 +1,14 @@
-var expect = require('chai').expect
-var fixMe = require('../lib/fix-me.js')
+var expect = require('chai').expect,
+    fs = require('fs'),
+    intercept = require("intercept-stdout"),
+    captured_text = "",
+    unhook_intercept = intercept(function(txt) {
```

Comment:
Sure, camel casing sounds good.

Happy to move into the specs themselves. Was thinking that these might be used in other specs in this file?